### PR TITLE
gh: add repo clone example

### PR DESCRIPTION
**Pull request by aaditkamat**
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/gh.md [hunk 1]
```diff
@@ -19,6 +19,10 @@
 
 `gh pr status`
 
+- Clone a repository locally:
+
+`gh repo clone {{OWNER/REPO}}`
```
Comment (navarroaxel) [inline pages/common/gh.md:24]: Please use lower case and with full name:
```suggestion
`gh repo clone {{owner}}/{{repository}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @aaditkamat, thanks for your submission. I left a comment.

Remember that you can create pages like `gh-alias.md`, `gh-clone.md`, etc. to explain every subcommand in detail.

**Review by owenvoke — APPROVED** [review]
LGTM after @navarroaxel's comment has been addressed.

I also agree that we can probably split `gh` up into sub-commands in future. 👍🏻

**Review by navarroaxel — APPROVED** [review]
Hey @aaditkamat, I applied the format fix to merge this PR. Thanks for your work!

**Comment by aaditkamat** [comment]
Thank you @navarroaxel and @owenvoke!

**Pull request merged**